src/gordon_gcp/plugins/janitor/authority.py

```diff
@@ -143,6 +143,9 @@ def __init__(self, config, metrics, crm_client, gce_client,
     async def _get_projects(self):
         projects = await self._get_active_project_ids()
         project_blacklist = set(self.config.get('project_blacklist', []))
+        project_whitelist = self.config.get('project_whitelist')
+        if project_whitelist:
+            projects &= set(project_whitelist)
```

Comment:
This is interesting, but I think the Janitor is still doing more work than it needs to. If someone adds projects to the whitelist, we could just skip the fetching of active project IDs altogether.

Reply:
I think project_whitelist is fine. If that happens, I think it's okay for the Janitor to fail, as long as we document how the parameter works in https://github.com/spotify/gordon-gcp/blob/develop/docs/config-janitor.rst

Reply:
I'll update the functionality and the doc 👍